badges.css

```diff
@@ -0,0 +1,40 @@
+:root {
+  font-size: 62.5%;
+}
+.badges-component {
+  display: flex;
+  justify-content: space-around;
+}
+.notification {
+  position: relative;
+  width: 5rem;
+  height: 5rem;
+}
+.icon-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 4rem;
+  height: 4rem;
+  color: #333333;
+  background: lavenderblush;
+  border: none;
+  outline: none;
+  border-radius: 50%;
+}
+.icon-img {
+  width: 100%;
+}
+.icon-badge-number {
+  position: absolute;
+  top: 0;
+  right: 0rem;
+  width: 1.5rem;
+  height: 1.5rem;
+  background: lightblue;
+  color: black;
```

Comment:
Define variables for colors and use them.

Reply:
donw with the suggestion